tests/foreman/ui/test_host.py

```diff
@@ -1027,289 +1023,6 @@ def test_positive_validate_inherited_cv_lce_ansiblerole(session, target_sat, mod
-            assert interface_result.execution
-
-
-@pytest.mark.tier2
-def test_global_registration_form_populate(
-    module_org,
-    session,
-    module_ak_with_cv,
-    module_lce,
-    module_promoted_cv,
-    default_architecture,
-    default_os,
-    target_sat,
-):
-    """Host registration form should be populated automatically based on the host-group
-
-    :id: b949e010-36b8-48b8-9907-36138342c72b
-
-    :expectedresults: Some of the fields in the form should be populated based on host-group
-        e.g. activation key, operating system, life-cycle environment, host parameters for
-        remote-execution, insights setup.
-
-    :CaseLevel: Integration
-
-    :steps:
-        1. create and sync repository
-        2. create the content view and activation-key
-        3. create the host-group with activation key, operating system, host-parameters
-        4. Open the global registration form and select the same host-group
-        5. check host registration form should be populated automatically based on the host-group
-
-    :BZ: 2056469, 1994654
-
-    :customerscenario: true
-    """
-    hg_name = gen_string('alpha')
-    iface = gen_string('alpha')
-    group_params = {'name': 'host_packages', 'value': constants.FAKE_0_CUSTOM_PACKAGE}
-    target_sat.api.HostGroup(
-        name=hg_name,
-        organization=[module_org],
-        lifecycle_environment=module_lce,
-        architecture=default_architecture,
-        operatingsystem=default_os,
-        content_view=module_promoted_cv,
-        group_parameters_attributes=[group_params],
-    ).create()
-    new_org = target_sat.api.Organization().create()
-    new_ak = target_sat.api.ActivationKey(organization=new_org).create()
-    with session:
-        session.hostgroup.update(
-            hg_name,
-            {
-                'activation_keys.activation_keys': module_ak_with_cv.name,
-            },
-        )
-        cmd = session.host.get_register_command(
-            {
-                'general.host_group': hg_name,
-                'advanced.rex_interface': iface,
-                'general.insecure': True,
-            },
-            full_read=True,
-        )
-        assert hg_name in cmd['general']['host_group']
-        assert module_ak_with_cv.name in cmd['general']['activation_key_helper']
-        assert constants.FAKE_0_CUSTOM_PACKAGE in cmd['advanced']['install_packages_helper']
-
-        session.organization.select(org_name=new_org.name)
-        cmd = session.host.get_register_command(
-            {
-                'general.organization': new_org.name,
-                'general.operating_system': default_os.title,
-                'general.insecure': True,
-            },
-            full_read=True,
-        )
-        assert new_org.name in cmd['general']['organization']
-        assert new_ak.name in cmd['general']['activation_keys']
-
-
 @pytest.mark.tier2
 def test_global_registration_with_capsule_host(
     session,
```

Comment:
Could you also move this test (or look for other tests) along with other global_registration tests?

Reply:
I feel this is more host/capsule related test than registration test.

Reply:
yes, this is definately a registration test against capsule

Reply:
Moved this to registration module.